docs/administration/Configuration.md

```diff
@@ -42,462 +42,2042 @@ ADMIN SET FRONTEND CONFIG ("key" = "value");
+
+##### label_clean_interval_second
+
+- **Default:** 14400
+- **Description:** The time interval at which labels are cleaned up. Unit: second. We recommend that you specify a short time interval to ensure that historical labels can be cleaned up in a timely manner.
+
+##### spark_dpp_version
+
+- **Default:** 1.0.0
+- **Description:** The version of Spark Dynamic Partition Pruning (DPP) used.
+
+##### spark_resource_path
+
+- **Default:** Empty string
+- **Description:** The root directory of the Spark dependency package.
+
+##### spark_launcher_log_dir
+
+- **Default:** sys_log_dir + "/spark_launcher_log"
+- **Description:** The directory that stores Spark log files.
+
+##### yarn_client_path
+
+- **Default:** StarRocksFE.STARROCKS_HOME_DIR + "/lib/yarn-client/hadoop/bin/yarn"
+- **Description:** The root directory of the Yarn client package.
+
+##### yarn_config_dir
+
+- **Default:** StarRocksFE.STARROCKS_HOME_DIR + "/lib/yarn-config"
+- **Description:** The directory that stores the Yarn configuration file.
+
+##### export_checker_interval_second
+
+- **Default:** 5
+- **Description:** The time interval at which load jobs are scheduled.
+
+##### export_task_pool_size
+
+- **Default:** 5
+- **Description:** The size of the unload task thread pool.
+
+#### Storage
+
+##### default_storage_medium
+
+- **Default:** HDD
+- **Description:** The default storage media that is used for a table or partition at the time of table or partition creation if no storage media is specified. Valid values: `HDD` and `SSD`. When you create a table or partition, the default storage media specified by this parameter is used if you do not specify a storage media type for the table or partition.
+
+##### tablet_sched_balancer_strategy
+
+- **Default:** disk_and_tablet
+- **Description:** The policy based on which load balancing is implemented among tablets. The alias of this parameter is `tablet_balancer_strategy`. Valid values: `disk_and_tablet` and `be_load_score`.
+
+##### tablet_sched_storage_cooldown_second
+
+- **Default:** -1
+- **Description:** The latency of automatic cooling starting from the time of table creation. The alias of this parameter is `storage_cooldown_second`. Unit: second. The default value `-1` specifies that automatic cooling is disabled. If you want to enable automatic cooling, set this parameter to a value greater than `-1`.
+
+##### tablet_stat_update_interval_second
+
+- **Default:** 300
+- **Description:** The time interval at which the FE retrieves tablet statistics from each BE. Unit: second.
+
+#### StarRocks shared-data cluster
+
+##### run_mode
+
+- **Default**: shared_nothing
+- **Description**: The running mode of the StarRocks cluster. Valid values: shared_data and shared_nothing (Default).
+
+shared_data indicates running StarRocks in shared-data mode.
+shared_nothing indicates running StarRocks in shared-nothing mode.
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/344dae45-4a5d-4ec0-bb3d-f8d837a7da9d)
I think it's better to organize this part as follows:
Valid values:
- `shared_data` indicates running StarRocks in shared-data mode.
- `shared_nothing` indicates running StarRocks in shared-nothing mode.